api/collections/serializers.py

```diff
@@ -94,3 +94,76 @@ def get_absolute_url(self, obj):
                 'node_link_id': obj._id
             }
         )
+
+
+class SingleLinkedNode(JSONAPIRelationshipsSerializer):
+    id = ser.CharField(source='node._id', required=False, allow_null=True)
```

Comment:
Right, this could be moved to ` api/base/serializers.py ` as ` RelationshipResourceIdentifierObject ` or something like that.